tests/component/test_stream_writers_do.py

```diff
@@ -431,6 +431,30 @@ def test___digital_multi_channel_writer___write_one_sample_multi_line___updates_
     assert di_multi_line_loopback_task.read() == datum
 
 
+def test___digital_multi_channel_writer___write_one_sample_multi_line_jagged___updates_output(
+    di_port0_loopback_task: nidaqmx.Task,
+    generate_task: Callable[[], nidaqmx.Task],
+    real_x_series_device: nidaqmx.system.Device,
+) -> None:
+    task = generate_task()
+    for port in real_x_series_device.do_ports:
+        task.do_channels.add_do_chan(
+            port.name,
+            line_grouping=LineGrouping.CHAN_FOR_ALL_LINES,
+        )
+    _start_do_task(task, is_port=True, num_chans=task.number_of_channels)
+    writer = DigitalMultiChannelWriter(task.out_stream)
+    num_channels = task.number_of_channels
+    samples_to_write = 256
+
+    # "sweep" up to the final value, the only one we'll validate
```

Comment:
FYI, 0xFF is not the most interesting value to validate. Consider stopping on 0xA5 or something.

Reply:
I pushed a change to update the final value to be 0xA5 instead of 255 for better validation.